src/Microsoft.Cci.Extensions/Writers/CSharp/CSDeclarationWriter.Methods.cs

```diff
@@ -264,13 +264,13 @@ private void WriteBaseConstructorCall(ITypeDefinition type)
 
             var ctors = baseType.Methods.Where(m => m.IsConstructor && _filter.Include(m));
```

Comment:
I think I would just filter out the ctors here instead of doing it in both places below. We already do other filtering here.